### PR TITLE
Ghetto cattle prod now works + code cleanup [READY]

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
fix: Cattleprod now properly stuns like before
code: Stun baton code is cleaner
/:cl:

[why]: Because it was driving me autistic.

### CODE REVIEW

File: hippiestation/code/game/objects/items/stunbaton.dm [hunk 1]
```diff
@@ -125,66 +94,18 @@
...
+	playsound(loc, 'sound/weapons/egloves.ogg', 50, 1, -1)
 
 	if(ishuman(L))
 		var/mob/living/carbon/human/H = L
 		H.forcesay(GLOB.hit_appends)
-
-	return 1
-
-/obj/item/melee/baton/attack(mob/M, mob/living/carbon/human/user)
-	if(status && user.has_trait(TRAIT_CLUMSY) && prob(50))
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/objects/items/stunbaton.dm:148]: Why are you removing this part?

**Comment by Manly-Dwarf** [comment]
How much have you tested this? Or is this another Guyon PR which is here to do nothing other than remove stuns instead of fixing the actual problems with stun combat?

File: hippiestation/code/game/objects/items/stunbaton.dm (hunk 1 above)
Comment (yoyobatty) [reply hippiestation/code/game/objects/items/stunbaton.dm:148]: because the code is already done in stunbaton under tg

**Comment by yoyobatty** [comment]
Nah, this is fully tested. Compare this with stunbaton tg code and you'll see what I did was remove the copypasted bs that was in our custom one.

**Comment by Manly-Dwarf** [comment]
thanks

**Comment by ghost** [comment]
+1

**Comment by yoyobatty** [comment]
This doesn't need +1's, it's a fix not a feature. All it needs is a code, check, approve and merge.

**Review by ghost — APPROVED** [review]

**Pull request merged**